recordedit/form.controller.js

```diff
@@ -104,6 +104,29 @@
             $log.info(response);
         }
 
+
+        /*
```

Comment:
Shouldn't this treatment be nearly universal for all column types?  It is mapping a blank user-entry to NULL which is the normal way of encoding "nothing provided" in the database.

If we don't treat all columns this way, I think it should be specific exceptions that **bypass** rather than having many cases fall through cracks